docker-compose.yml

```diff
@@ -49,6 +49,7 @@ services:
   gateway:
     image: "${CONTAINER_REGISTRY}/gateway:${GATEWAY_TAG}"
     depends_on:
+      - provisioner
```

Comment:
We probably don't want this :smile:

Reply:
We had it on main so changing it seemed unrelated to this PR. Perhaps it's because the gateway will start deployers, and the deployers will need a provisioner connection?